composer/trainer/trainer.py

```diff
@@ -946,6 +974,17 @@ def __init__(
                     stream=console_stream,
                 ))
 
+        if checkpoint_save_path is not None:
+            object_store_logger = _create_object_store_logger_from_uri(uri=checkpoint_save_path)
+            if object_store_logger:
+                loggers.append(object_store_logger)
```

Comment:
So what happens if I specify a `WandBLogger` and use W&B for checkpointing? Do I get two `WandBLoggers` in this list and is that a problem?

Reply:
It doesn't seem to cause a problem because wandb ignores calls to `wandb.log` with the exact same key and step, but we could add some code to de-duplicate loggers.

Reply:
Hm, perhaps if the user specifies a wandblogger separately, we just use that logger for checkpoint things as well rather than creating a new one and potentially duplicating calls to wandb. Is that possible/reasonable?

Reply:
done